qiskit_ibm_runtime/fake_provider/fake_backend.py

```diff
@@ -31,15 +31,15 @@
 from qiskit.utils import optionals as _optionals
 from qiskit.transpiler import Target
 from qiskit.providers import Options
-from qiskit.providers.fake_provider.utils.json_decoder import (
-    decode_backend_configuration,
-    decode_backend_properties,
-    decode_pulse_defaults,
-)
 
 from qiskit.providers.basic_provider import BasicSimulator
 
 from qiskit_ibm_runtime.utils.backend_converter import convert_to_target
+from qiskit_ibm_runtime.utils.backend_decoder import (
+    _decode_backend_configuration,
```

Comment:
Can we rename the function to not start with `_` if it's to be imported and used elsewhere?